xchange-coinbasepro/src/main/java/org/knowm/xchange/coinbasepro/CoinbasePro.java

```diff
@@ -218,18 +218,18 @@ CoinbaseProSendMoneyResponse sendMoney(
       @QueryParam("after") Integer startingOrderId)
       throws CoinbaseProException, IOException;
 
-  @GET
-  @Path("accounts/{account_id}/transfers")
-  @Consumes(MediaType.APPLICATION_JSON)
-  CoinbaseProTransfers transfers(
-      @HeaderParam("CB-ACCESS-KEY") String apiKey,
-      @HeaderParam("CB-ACCESS-SIGN") ParamsDigest signer,
-      @HeaderParam("CB-ACCESS-TIMESTAMP") SynchronizedValueFactory<Long> nonce,
-      @HeaderParam("CB-ACCESS-PASSPHRASE") String passphrase,
-      @PathParam("account_id") String accountId,
-      @QueryParam("profile_id") String profileId,
-      @QueryParam("limit") Integer limit,
-      @QueryParam("after") String createdAtDate);
+//  @GET
+//  @Path("accounts/{account_id}/transfers")
+//  @Consumes(MediaType.APPLICATION_JSON)
+//  CoinbaseProTransfers transfers(
+//      @HeaderParam("CB-ACCESS-KEY") String apiKey,
+//      @HeaderParam("CB-ACCESS-SIGN") ParamsDigest signer,
+//      @HeaderParam("CB-ACCESS-TIMESTAMP") SynchronizedValueFactory<Long> nonce,
+//      @HeaderParam("CB-ACCESS-PASSPHRASE") String passphrase,
+//      @PathParam("account_id") String accountId,
+//      @QueryParam("profile_id") String profileId,
+//      @QueryParam("limit") Integer limit,
+//      @QueryParam("after") String createdAtDate);
```

Comment:
I assume this will be removed completely right?

Reply:
Yes. I think this was before I pinged you guys about this.